src/views/ValidatorDetail.vue

```diff
@@ -113,12 +121,13 @@ const DPOSStore = createNamespacedHelpers('DPOS')
       'setErrorMsg'
     ]),
     ...DPOSStore.mapActions([
-      'redelegateAsync'
+      'redelegateAsync',
+      'consolidateDelegations',
     ]),
     ...DappChainStore.mapActions([
       'getValidatorsAsync',
       'claimRewardAsync',
-      'getDpos2'
+      'getDpos3'
```

Comment:
Can we extract all strings like this to a `constants.ts` file and use that? If we want in the future to use `dpos4` for example, it'd be nicer to just modify a constant in a file, than have to replace it everywhere.

Reply:
Actually I have a plan for that. No strings but real function signatures. Check out https://github.com/mrcrowl/vuex-typex